### PR TITLE
round/floor/trunc etc functions for decimals on n digit

### PR DESCRIPTION
**Pull request by codingstill**
Hi Teo, 

Have a look and let me know if you need any changes (syntax, structure, etc).

### CODE REVIEW

**Comment by thodorisbais** [comment]
Super happy to merge this, Taso, thanks a ton! 🙏 🎉

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "numeric"
   ],
   "scripts": {
-    "test": "istanbul cover  jasmine-node spec"
+    "test": "istanbul cover jasmine-node spec",
+    "wntest": "istanbul cover node_modules/jasmine-node-xt/bin/jasmine-node spec"
```
Comment (thodorisbais) [inline package.json]: Btw, @codingstill , what is the purpose of this script?

Comment (thodorisbais) [reply package.json]: @codingstill I mean, I feel this is redundant, given that we defined `jasmine-node-xt` in our `devDependencies` here. Am I missing something?

Comment (codingstill) [reply package.json]: Hi Teo!
`npm run test` gives an error in Windows, so I added `npm run wntest` to be able to run the tests.

Comment (thodorisbais) [reply package.json]: Cool, good to know, thanks!

**Pull request merged**